django/middleware/csrf.py

```diff
@@ -136,6 +138,35 @@ class CsrfViewMiddleware(MiddlewareMixin):
     This middleware should be used in conjunction with the {% csrf_token %}
     template tag.
     """
+    @cached_property
+    def csrf_trusted_origins_hosts(self):
```

Comment:
Should these properties be marked private to allow tweaking their return value over time? Their use is mainly for optimization purposes.

Reply:
It's not documented so it's not considered stable API, even if not prefixed with an underscore.